manage

```diff
@@ -611,7 +611,7 @@ elif [ "$1" == 'install' ] || [ "$1" == 'uninstall' ];then
   status "${action^}ing \e[1m${app}\e[0m\e[96m..." | tee -a "$logfile"
   echo
   cd $HOME
-  if [[ "$3" == "update" ]]; then
+  if [ "$3" == "update" ]; then
```

Comment:
enlighten me. what is the difference between the [[ ]] and [ ]. I know the former is more versatile but thats about it. if you have some resource that explains (well) that would be nice

Reply:
I don't have a particularly good source on-hand, but IIRC, the only time double brackets are necessary is for advanced operations like partial matches, regex, `=~`, things like that. Simple `==` or `-gt` don't need double brackets.

Reply:
https://unix.stackexchange.com/questions/306111/what-is-the-difference-between-the-bash-operators-vs-vs-vs

Reply:
@theofficialgman 
> https://unix.stackexchange.com/questions/306111/what-is-the-difference-between-the-bash-operators-vs-vs-vs